lib/nest-thermostat-accessory.js

```diff
@@ -383,21 +383,15 @@ NestThermostatAccessory.prototype.setTargetTemperatureDebounced = debounce(funct
   return promise.then(() => this.updateTargetWithCorrectUnitsAsync(setting, targetTemperature, "target temperature")).asCallback(callback);
 }, 5000);
 
-NestThermostatAccessory.prototype.setHeatCoolIfNeeded = function () {
-  return this.device.hvac_mode === 'heat-cool'
-    ? Promise.resolve()
-    : this.updateDevicePropertyAsync("hvac_mode", 'heat-cool', "target heating cooling");
-};
-
 NestThermostatAccessory.prototype.setCoolingThresholdTemperatureDebounced = debounce(function (targetTemperature, callback) {
-  return this.setHeatCoolIfNeeded()
-    .then(() => this.updateTargetWithCorrectUnitsAsync('target_temperature_high_', targetTemperature, "cooling threshold temperature"))
+  return Promise.resolve()
+    .then(() => this.updateTargetWithCorrectUnitsAsync('target_temperature_', targetTemperature, "cooling threshold temperature"))
```

Comment:
I think this needs to be target_temperature_high_ for those who *do* use auto mode. (ditto below with target_temperature_low_)

Reply:
yes, i just discovered that. revising.